filesystem.go

```diff
@@ -35,3 +35,21 @@ func fileExists(path string) bool {
 
 	return true
 }
+
+func fileExistsAndIsDirectory(dirPath string) bool {
```

Comment:
you shouldnt need these functions.  The directory is already created and written to by other code.